spynnaker/pyNN/models/spike_source/spike_source_array_vertex.py

```diff
@@ -166,10 +166,6 @@ def clear_spike_recording(self, buffer_manager, placements, graph_mapper):
     def set_model_max_atoms_per_core(new_value=sys.maxsize):
         SpikeSourceArrayVertex._model_based_max_atoms_per_core = new_value
```

Comment:
_model_based_max_atoms_per_core  does not exist inside the class to begin with. so broken if i were to put sockets on here

Reply:
This is the same as master so out of scope for this PR.